plugins/woocommerce-blocks/assets/js/types/type-defs/hooks.ts

```diff
@@ -69,3 +69,38 @@ export type Query = {
 	orderby: string;
 	order: string;
 };
+
+export type RatingValues = 0 | 1 | 2 | 3 | 4 | 5;
+
+type AttributeCount = {
```

Comment:
Nitpick: it can be used in Filter by Attribute:
https://github.com/woocommerce/woocommerce/blob/30cc92fddc42277a3e7d1ce26e5a044e72265980/plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/types.ts#L27

Reply:
nice one. addressed here cb95de78a2fb6aa167c15a89f049e28511e53137